.env.example

```
@@ -28,4 +28,9 @@ KAKAO_CLIENT_SECRET=your_kakao_client_secret
 JWT_SECRET=your_very_long_and_secret_random_string_for_signing_tokens_at_least_64_characters
 
 # 6. Frontend Redirect URL
-OAUTH2_REDIRECT_URL=http://localhost:3000/oauth2/redirect
\ No newline at end of file
+OAUTH2_REDIRECT_URL=http://localhost:3000/oauth2/redirect
+
+# 7. SMS
+SMS_API_KEY = your_sms_api_key
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)